r/R/trackplots.R

```diff
@@ -382,7 +390,7 @@ trackplot_coverage <- function(fragments, region, groups,
   }
   colors <- colors[seq_len(ncol(membership_matrix))]
 
-  bin_centers <- seq(region$start, region$end - 1, region$tile_width) + region$tile_width / 2
+  bin_centers <- seq(region$start, region$end - 1, region$tile_width) + region$tile_width %/% 2
```

Comment:
Looking back at this, I think the original calculation was at least off by 0.5 (i.e. a width-1 bin shouldn't be shifted, and a width-2 bin should be ideally shifted by 0.5bp). Based on that I'd propose `(region$tile_width-1) / 2`.

Though if you think the off-integer placement looks bad keeping the `%/%` is a good way to solve that.

Reply:
It was moreso in the case of having tile width of 1, where region < 500.  I looked at the vector and it would always be duplicates with additions of 1 (ie c(10, 10, 12, 12, 14, 14)). Therefore we lost the bin centers for half of the vectors.  Your solution overall works better I think though, but I would make sure that bin_centers isn't being outputted as an integer list.